crates/re_viewport/src/viewport_blueprint.rs

```diff
@@ -265,10 +268,16 @@ impl ViewportBlueprint {
 
             Item::DataResult(space_view_id, instance_path) => {
                 self.space_view(space_view_id).map_or(false, |space_view| {
-                    space_view
-                        .contents
-                        .entity_path_filter
-                        .is_included(&instance_path.entity_path)
+                    let entity_path = &instance_path.entity_path;
+
+                    // TODO(#5742): including any path that is—or descend from—the space origin is
```

Comment:
I mentioned it in the issue itself, but yeah, this is a fragile partial re-implementation of the query result evaluation. Let's just use the query results directly instead of trying to replicate the logic.

Reply:
Yup, I've laid out the full action plan in https://github.com/rerun-io/rerun/issues/5742